interaction_lib.py

```
@@ -0,0 +1,307 @@
+        return "<p>No interaction terms to interpret.</p>"
+    
+    effect_name = "Odds Ratio" if model_type == 'logit' else "Incidence Rate Ratio"
+    
+    sig_interactions = [k for k, v in results.items() if v['significant']]
+    
+    if not sig_interactions:
+        return f"""<div class='alert alert-info'>
+            <b>No Significant Interactions Found</b><br>
+            The effect of predictors does not significantly vary across levels of other variables.
+            Main effects can be interpreted independently.
+        </div>"""
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)